plugins/logging.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const good = require('@hapi/good');
```

Comment:
I think this module is getting EOLed in December

> Note: this module is being deprecated on December 31st, 2020 due to lack to available support resources. Please consider using another logging plugin.

https://www.npmjs.com/package/@hapi/good

Should we use https://github.com/felixheck/laabr or https://www.npmjs.com/package/morgan?

Reply:
Yes, I have laabr added, tried to keep the request, response logging format across SD API and QS same for easier log searching and debugging. We can move to new lib across both systems together.